test/e2e/testdata/backend-dualstack.yaml

```diff
@@ -0,0 +1,95 @@
+apiVersion: v1
+kind: Service
```

Comment:
should this be removed since its being tested in `httproute-dualstack.yaml` ?

Reply:
Yes I think it can be remove.